parts/linux/cloud-init/artifacts/cse_cmd.sh

```diff
@@ -12,7 +12,7 @@ done;
 {{GetInitAKSCustomCloudFilepath}} >> /var/log/azure/cluster-provision.log 2>&1;
 {{end}}
 ADMINUSER={{GetParameter "linuxAdminUsername"}}
-CONTAINERD_VERSION={{GetParameter "containerdVersion"}}
+CONTAINERD_VERSION="1.3.4"
 MOBY_VERSION={{GetParameter "mobyVersion"}}
```

Comment:
can you do the same for the moby version ? in next PR?

Reply:
yea ill make a separate pr for that